components/Menu/structure.ts

```diff
@@ -18,218 +20,231 @@ const menu: MenuCategoryProps[] = [
   {
     title: "Products",
     description: "Teleport Products",
-    href: "/teleport/",
+    href: `https://${HOST}/`,
```

Comment:
What is the reasoning for adding the full host in this file?

Reply:
Removed